controller/linodemachine_controller.go

```diff
@@ -310,41 +317,48 @@ func (r *LinodeMachineReconciler) reconcileCreate(
 	return linodeInstance, nil
 }
 
-func (r *LinodeMachineReconciler) configureDisksControlPlane(
+func (r *LinodeMachineReconciler) configureInstanceProfile(
```

Comment:
We might want to keep in `ControlPlane` since this is just for control plane nodes